x-pack/plugins/infra/public/alerting/metric_threshold/components/expression.tsx

```diff
@@ -387,31 +382,6 @@ export const Expressions: React.FC<Props> = (props) => {
             checked={ruleParams.alertOnNoData}
             onChange={(e) => setRuleParams('alertOnNoData', e.target.checked)}
           />
-          <EuiCheckbox
-            id="metrics-alert-partial-buckets-toggle"
-            label={
-              <>
-                {i18n.translate('xpack.infra.metrics.alertFlyout.shouldDropPartialBuckets', {
```

Comment:
@simianhacker I suggest we start using rules instead of alerts whenever we refer to rules `xpack.infra.metrics.ruleFlyout.shouldDropPartialBuckets`